requirements.txt

```diff
@@ -1,21 +1,17 @@
-# Pin these requirements to the latest versions when you create a new project,
-# e.g. Django==3.0.2.
-Django
-psycopg2-binary
-gunicorn
-dj-database-url
-whitenoise
-django-compressor
-sentry-sdk
-django-money
-django-datatables-view
+Django==3.1.7
```

Comment:
I pinned everything here by opening a console against the production app and running `pip freeze` to generate the current versions of our dependencies on the live site.